Clarify Default Chat API Key creation for existing instances (#3334)
meilisearch/documentation: learn/chat/getting_started_with_chat.mdx

## Find your chat API key

> When Meilisearch runs with a master key on a **fresh instance**, it automatically creates a "Default Chat API Key" with `chatCompletions` permission.
> 
> <Note>
> The Default Chat API Key is only created automatically for new Meilisearch instances. If you have an existing instance, you'll need to create your own API key with `chatCompletions` permission.
> </Note>
> 
> Find existing keys using:

Reviewer: ```suggestion
When Meilisearch runs with a master key on an instance created after v1.15.1, it automatically generates a "Default Chat API Key" with `chatCompletions` and `search` permissions on all indexes. Check if you have the key using:
```

```bash
curl http://localhost:7700/keys \
  -H "Authorization: Bearer MEILISEARCH_KEY"
```

> Look for the key with "Default Chat API Key" in the description. Use this key when querying the `/chats` endpoint.

Reviewer: ```suggestion
Look for the key with the description "Default Chat API Key" Use this key when querying the `/chats` endpoint.
```

> ### Creating a chat API key for existing instances
> 
> If you have an existing Meilisearch instance and don't see a Default Chat API Key, create one manually:
> 
> ```bash
> curl \
>   -X POST 'http://localhost:7700/keys' \
>   -H 'Authorization: Bearer MEILISEARCH_KEY' \
>   -H 'Content-Type: application/json' \
>   --data-binary '{
>     "name": "Chat API Key",
>     "description": "API key for chat completions",
>     "actions": ["search", "chatCompletions", "documents.get"],
>     "indexes": ["*"],
>     "expiresAt": null
>   }'
> ```
> 
> This creates an API key with `search`, `chatCompletions`, and `documents.get` permissions, which are required for the chat feature to search and retrieve document content.

Reviewer: ````suggestion
### Troubleshooting: Missing default chat API key

If your instance does not have a Default Chat API Key, create one manually:

```bash
curl \
  -X POST 'http://localhost:7700/keys' \
  -H 'Authorization: Bearer MEILISEARCH_KEY' \
  -H 'Content-Type: application/json' \
  --data-binary '{
    "name": "Chat API Key",
    "description": "API key for chat completions",
    "actions": ["search", "chatCompletions"],
    "indexes": ["*"],
    "expiresAt": null
  }'
```
````
